packages/contracts/contracts/UnirepApp.sol

```diff
@@ -282,4 +293,45 @@ contract UnirepApp {
     ) public view returns (bool) {
         return dataVerifier.verifyProof(publicSignals, proof);
     }
+
+    /**
+     * Claim the report positive reputation
+     * @param publicSignals: public signals
+     * @param proof: epochKeyProof
+     * @param change: reputation score
+     */
+    function claimReportPosRep(
+        uint256[] calldata publicSignals,
+        uint256[8] calldata proof, // epochKeyProof
+        uint256 change
+    ) public {
+		// check if proof is used before
+        bytes32 nullifier = keccak256(abi.encodePacked(publicSignals, proof));
+        if (proofNullifier[nullifier]) {
+            revert ProofHasUsed();
+        }
+        proofNullifier[nullifier] = true;
+
+        EpochKeyVerifierHelper.EpochKeySignals memory signals = epkHelper
```

Comment:
This should be changed using `VerifierHelperManager` to decode and verify the proof. After #406 is merged, this can be added into the contract